cmd/ratify/cmd/verify.go

```diff
@@ -73,6 +75,15 @@ func verify(opts verifyCmdOptions) error {
 		return errors.New("subject parameter is required")
 	}
 
+	subRef, err := utils.ParseSubjectReference(opts.subject)
+	if err != nil {
+		return err
+	}
+
+	if subRef.Digest == "" {
+		fmt.Println("Warning: Tagged references should NOT be used. The resolved digest may not point to the same signed artifact, since tags are mutable.")
```

Comment:
`Warning: Tagged references should NOT be used. The resolved digest may not point to the same signed artifact, since tags are mutable.` warning message is used in 3 different places. Make this a constant and reuse across packages?